packages/mgt-chat/src/utils/onRenderMessage.tsx

```diff
@@ -54,7 +57,28 @@ const MgtAdaptiveCard = (msg: MgtAdaptiveCardProps) => {
   const defaultOnRender = msg?.defaultOnRender;
   const messageProps = msg.messageProps;
   const defaultRender = defaultOnRender ? defaultOnRender(messageProps) : <></>;
-  return adaptiveCardAttachments.length ? <div ref={cardRef}></div> : defaultRender;
+  const Container = messageContainer(msg.messageProps.message);
+  const author = isChatMessage(msg.messageProps.message) ? msg.messageProps.message?.senderDisplayName : '';
+  const timestamp = onDisplayDateTimeString(msg.messageProps.message.createdOn);
+
+  return adaptiveCardAttachments.length ? (
+    <Container author={author} timestamp={timestamp}>
```

Comment:
Can we add a details slot to show an edited badge if the message was edited?

Reply:
I'm using the `message.status` property value for the details. Is this the correct value?